daprdocs/content/en/reference/components-reference/supported-middleware/middleware-wasm.md

```diff
@@ -32,15 +40,15 @@ spec:
 
 ## Spec metadata fields
 
-| Field | Details | Example |
-|-------|---------|---------|
-| path | A relative or absolute path to the WASM module | "./hello.wasm" |
-| runtime | The WASM runtime of your WASM binary. Only `wazero` is supported | ["wazero"](https://github.com/tetratelabs/wazero) |
-
+| Field    | Details                                        | Example        |
+|----------|------------------------------------------------|----------------|
+| path     | A relative or absolute path to the Wasm binary | "./hello.wasm" |
+| poolSize | The instance count of the Wasm binary          | 10             |
```

Comment:
Why did you remove the `runtime` property from this table? Was this unintentional?

Reply:
Also can you add the `poolsize` property into the YAML example.

Reply:
> Why did you remove the runtime property from this table? Was this unintentional?

This was intentional because the only other runtimes that could work are build flags, not runtime flags, as they would require CGO (and are build flags in wapc-go). The first impl only supports one runtime, wazero as there's no plan to optimizing towards CGO per the below thread.

https://github.com/dapr/components-contrib/pull/1833#issuecomment-1186051640

> Also can you add the poolsize property into the YAML example.

sure can do